otsdaq/TableCore/TableView.cc

```diff
@@ -2888,7 +2892,7 @@ int TableView::fillFromCSV(const std::string& data, const int& dataOffset, const
 		// if row was modified, assign author and timestamp
 		if(author != "" && rowWasModified)
 		{
-			__COUT__ << "Row=" << (int)r << " was modified!" << __E__;
+			__COUT_TYPE__(TLVL_DEBUG+20) << "Row=" << (int)r << " was modified!" << __E__;
```

Comment:
I'm increasingly of the opinion that these macros do not help, and make things harder to understand. Is there any reason to not change to TLOG() throughout?